src/views/Downloads.vue

```diff
@@ -139,6 +148,24 @@
     watch: {
       $route: 'updateData'
     },
+    computed: {
+      sponsor: function () {
+        let random = Math.floor(Math.random() * Math.floor(2));
+        if(random === 0) {
+          return {
+            src: window.outerWidth >= 576 ?
+                    "/assets/images/zaphosting-large.png" :
+                    "/assets/images/zaphosting-small.jpg",
+            href: "https://zap-hosting.com/sponge"
```

Comment:
I'd prefer not hard-coding this here, but rather have something like `sponsors.json` where they are all defined. Not entirely sure about the conditional image.

Reply:
The conditional image was requested by multiply people. Feel free to change the way it works.